test/team_stats_test.rb

```diff
@@ -88,7 +88,8 @@ def setup
     @team_23 = Team.new(team_23_info)
     @team_52 = Team.new(team_52_info)
 
-    @teams = [
```

Comment:
I think it's fine to keep the bracket on the same line as the @teams =

Reply:
Sure, doesn't matter